### PR TITLE
#3817 Added the journal summary in between header and announcements

### PR DESCRIPTION
**Pull request by thinkbulecount2**
[See issue #3817: Add option to display journal summary on homepage](https://github.com/pkp/pkp-lib/issues/3817)

### CODE REVIEW

File: templates/frontend/pages/indexJournal.tpl [hunk 1]
```diff
@@ -28,6 +28,12 @@
 		</div>
 	{/if}
 
+    {* Journal Description *}
```
Comment (NateWr) [inline templates/frontend/pages/indexJournal.tpl]: We use tab indents. You should be able to update your editor to use tab indents and automatically format the code on save.

File: templates/frontend/pages/indexJournal.tpl [hunk 2]
```diff
@@ -28,6 +28,12 @@
...
+    {if $currentContext->getLocalizedData('description')}
+        <h2>{translate key="about.aboutContext"}</h2>
```
Comment (NateWr) [inline templates/frontend/pages/indexJournal.tpl]: Let's wrap this whole section in a div:

```
<div class="homepage_about">
  ...
</div>
```

Journals often use CSS code to modify default theme, so wrapping it in a div with a unique class makes it easier for them to target the description for changing the style if they want.

When you do this, you'll notice the margins between the description and the announcements gets collapsed. You'll probably need to add some CSS code to get the spacing correct. You can add that to `plugins/themes/default/styles/pages/indexJournal.less`.

(Less is similar to Sass, with a slightly different syntax. You can read up on it at: http://lesscss.org/)

File: templates/frontend/pages/indexJournal.tpl (hunk 1 above)
Comment (thinkbulecount2) [reply templates/frontend/pages/indexJournal.tpl]: I've got 4 tab indents set -- do you mind double-checking mine to make sure I got that one right? :) Sometimes it doesn't change or appear as tabs/spaces.

File: templates/frontend/pages/indexJournal.tpl (hunk 2 above)
Comment (thinkbulecount2) [reply templates/frontend/pages/indexJournal.tpl]: Got that done, plus the theme option!

File: plugins/themes/default/styles/pages/indexJournal.less [hunk 3]
```diff
@@ -39,6 +39,15 @@
...
 
+	.homepage_about {
+		padding-top: @triple;
+		padding-bottom: @triple;
+
+		h2 {
+			margin-top: -@base
+		}
+	}
+
```
Comment (NateWr) [inline plugins/themes/default/styles/pages/indexJournal.less]: :+1:

**Pull request merged**